test/Scripts.Integration.Test/SmokeTester.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.IO;
+using UnityEngine;
+
+public class SmokeTester : MonoBehaviour
```

Comment:
This can be confusing with the actual Sentry smoke.
The script below that Deletes it before adding a new one is confusing. Best to rename this file to avoid confusion